### PR TITLE
dev/reference, v3.0/reference, v2.1/reference: add the downstream permission for lightning

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

TiDB Lightning will require the permission in downstream TiDB while only FAQ introduce it and deploy doc has no mention about it.
So I add the downstream permission in the Lightning deploy doc.
Also change the name from TiDB-Lightning to TiDB Lightning in documents.

### What is the related PR or file link(s)? 

Apply in dev, v3.0, v2.1

### Checklist 

- [ ] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by ericsyh** [comment]
@kennytm Added the admin checksum permission in https://github.com/pingcap/docs-cn/pull/1652/commits/9ed06e3cdb9808c0a1e44fe8299d9ebd4c41bfb8

**Comment by ericsyh** [comment]
@yikeke Resubmit the PR of https://github.com/pingcap/docs-cn/pull/1629

File: dev/faq/tidb-lightning.md [hunk 1]
```diff
@@ -23,12 +23,14 @@ Lightning 需要以下权限：
...
 * DROP
 
-另外，存储断点的数据库额外需要以下权限：
+存储断点的数据库额外需要以下权限：
 
 * INSERT
 * DELETE
 
-## Lightning 在导数据过程中某个表报错了，会影响其他表吗？进程会马上退出吗？
+如果 TiDB Lightning 配置项 `checksum = true`，需要下游 TiDB admin 用户权限。
```
Comment (yikeke) [inline dev/faq/tidb-lightning.md]:
```suggestion
如果 TiDB Lightning 配置项 `checksum = true`，则下游 TiDB 需要有 admin 用户权限。
```

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
 
-- TiDB-Lightning 运行后，TiDB 集群将无法正常对外提供服务。
+- TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。
 - 若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致迟延 (stall)。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
 
     ```sh
     bin/tidb-lightning-ctl -switch-mode=normal
     ```
 
+- TiDB Lightning 需要下游 TiDB 的权限：
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- TiDB Lightning 需要下游 TiDB 有如下权限：
```

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 3]
```diff
@@ -1,23 +1,37 @@
...
+  |----:|:------|
+  | SELECT | Tables |
+  | INSERT | Tables |
+  | UPDATE | Tables |
+  | DELETE | Tables |
+  | CREATE | Databases, tables |
+  | DROP | Databases, tables |
+  | ALTER | Tables |
+  
+  如果 TiDB Lightning 配置项 `checksum = true`，需要下游 TiDB admin 用户权限。
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/deployment.md]:
```suggestion
  如果 TiDB Lightning 配置项 `checksum = true`，则下游 TiDB 需要有 admin 用户权限。
```

File: v2.1/reference/tools/tidb-lightning/deployment.md [hunk 4]
```diff
@@ -1,23 +1,37 @@
...
 
-- TiDB-Lightning 运行后，TiDB 集群将无法正常对外提供服务。
+- TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。
 - 若 `tidb-lightning` 崩溃，集群会留在“导入模式”。若忘记转回“普通模式”，集群会产生大量未压缩的文件，继而消耗 CPU 并导致迟延 (stall)。此时，需要使用 `tidb-lightning-ctl` 手动将集群转回“普通模式”：
 
     ```sh
     bin/tidb-lightning-ctl -switch-mode=normal
     ```
 
+- TiDB Lightning 需要下游 TiDB 的权限：
```
Comment (yikeke) [inline v2.1/reference/tools/tidb-lightning/deployment.md]:
```suggestion
- TiDB Lightning 需要下游 TiDB 有如下权限：
```

File: v2.1/reference/tools/tidb-lightning/deployment.md [hunk 5]
```diff
@@ -1,23 +1,37 @@
...
+  |----:|:------|
+  | SELECT | Tables |
+  | INSERT | Tables |
+  | UPDATE | Tables |
+  | DELETE | Tables |
+  | CREATE | Databases, tables |
+  | DROP | Databases, tables |
+  | ALTER | Tables |
+  
+  如果 TiDB Lightning 配置项 `checksum = true`，需要下游 TiDB admin 用户权限。
```
Comment (yikeke) [inline v2.1/reference/tools/tidb-lightning/deployment.md]:
```suggestion
  如果 TiDB Lightning 配置项 `checksum = true`，则下游 TiDB 需要有 admin 用户权限。
```

File: dev/faq/tidb-lightning.md (hunk 1 above)
Comment (ericsyh) [reply dev/faq/tidb-lightning.md]: How about "TiDB Lightning 需要有下游 TiDB admin 用户权限" ?
"则下游 TiDB 需要有 admin 用户权限 " and "则下游 TiDB 用户需要有 admin 权限"  i think may not so accurate.

Comment (yikeke) [reply dev/faq/tidb-lightning.md]: "TiDB Lightning 需要有下游 TiDB admin 用户权限" sounds good to me. Thanks.

File: v2.1/faq/tidb-lightning.md [hunk 6]
```diff
@@ -23,12 +23,14 @@ Lightning 需要以下权限：
...
 * DROP
 
-另外，存储断点的数据库额外需要以下权限：
+存储断点的数据库额外需要以下权限：
 
 * INSERT
 * DELETE
 
-## Lightning 在导数据过程中某个表报错了，会影响其他表吗？进程会马上退出吗？
+如果 TiDB Lightning 配置项 `checksum = true`，需要下游 TiDB admin 用户权限。
```
Comment (yikeke) [inline v2.1/faq/tidb-lightning.md]:
```suggestion
如果 TiDB Lightning 配置项 `checksum = true`，则 TiDB Lightning 需要有下游 TiDB admin 用户权限。
```

File: v3.0/faq/tidb-lightning.md [hunk 7]
```diff
@@ -24,12 +24,14 @@ Lightning 需要以下权限：
...
 * DROP
 
-另外，存储断点的数据库额外需要以下权限：
+存储断点的数据库额外需要以下权限：
 
 * INSERT
 * DELETE
 
-## Lightning 在导数据过程中某个表报错了，会影响其他表吗？进程会马上退出吗？
+如果 TiDB Lightning 配置项 `checksum = true`，需要下游 TiDB admin 用户权限。
```
Comment (yikeke) [inline v3.0/faq/tidb-lightning.md]:
```suggestion
如果 TiDB Lightning 配置项 `checksum = true`，则 TiDB Lightning 需要有下游 TiDB admin 用户权限。
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 8]
```diff
@@ -1,24 +1,38 @@
...
+  |----:|:------|
+  | SELECT | Tables |
+  | INSERT | Tables |
+  | UPDATE | Tables |
+  | DELETE | Tables |
+  | CREATE | Databases, tables |
+  | DROP | Databases, tables |
+  | ALTER | Tables |
+  
+  如果 TiDB Lightning 配置项 `checksum = true`，需要下游 TiDB admin 用户权限。
```
Comment (yikeke) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
  如果 TiDB Lightning 配置项 `checksum = true`，则 TiDB Lightning 需要有下游 TiDB admin 用户权限。
```

**Pull request merged**